indico/migrations/versions/20210630_1104_356b8985ae7c_add_survey_anonymous_submissions_table.py

```diff
@@ -0,0 +1,32 @@
+"""Add anonymous_submissions table for surveys
+
+Revision ID: 356b8985ae7c
+Revises: 90384b9b3d22
+Create Date: 2021-06-30 11:04:32.349228
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '356b8985ae7c'
+down_revision = '90384b9b3d22'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'anonymous_submissions',
+        sa.Column('survey_id', sa.Integer(), nullable=False, index=True),
+        sa.Column('user_id', sa.Integer(), nullable=False, index=True),
+        sa.ForeignKeyConstraint(['survey_id'], ['event_surveys.surveys.id']),
+        sa.ForeignKeyConstraint(['user_id'], ['users.users.id']),
```

Comment:
```suggestion
        sa.ForeignKeyConstraint(['survey_id'], ['event_surveys.surveys.id'], ondelete='CASCADE'),
        sa.ForeignKeyConstraint(['user_id'], ['users.users.id'], ondelete='CASCADE'),
```

db_diff never lies :D

```sql
[adrian@eluvian:~/dev/indico/py3/src:(remotes/javier/keep-surveys-anonymous)]> python ./bin/utils/db_diff.py -v
** createdb -T indico_template indico_dbdiff
** indico db prepare
** Calculating differences
** dropdb indico_dbdiff
** pygmentize -l sql -f terminal256 -O style=native,bg=dark
alter table "event_surveys"."anonymous_submissions" drop constraint "fk_anonymous_submissions_survey_id_surveys";

alter table "event_surveys"."anonymous_submissions" drop constraint "fk_anonymous_submissions_user_id_users";

alter table "event_surveys"."anonymous_submissions" add constraint "fk_anonymous_submissions_survey_id_surveys" FOREIGN KEY (survey_id) REFERENCES event_surveys.surveys(id) ON DELETE CASCADE;

alter table "event_surveys"."anonymous_submissions" add constraint "fk_anonymous_submissions_user_id_users" FOREIGN KEY (user_id) REFERENCES users.users(id) ON DELETE CASCADE;
```

Reply:
Ah, yup. I added those on the original file. But forgot to re-add them when changing the class name in a previous suggestion! Fixed.